addons/mail/static/src/js/chatter.js

```diff
@@ -106,6 +106,12 @@ var Chatter = Widget.extend({
         var def = this._dp.add(Promise.all(fieldDefs));
         this._render(def).then(this._updateMentionSuggestions.bind(this));
 
+        // Drag-Drop files
+        // Allowing body to detect dragenter and dragleave for display
+        var $body = $('body');
+        $body.on('dragleave', this._onBodyFileDragLeave.bind(this));
```

Comment:
when you add event listener in `start()`, you should also remove them in `destroy()`

Reply:
Done